docs/docs/aztec/concepts/pxe/index.md

```diff
@@ -8,44 +8,101 @@ importance: 1
 
 import Image from "@theme/IdealImage";
 
-The Private Execution Environment (or PXE, pronounced 'pixie') is a client-side library for the execution of private operations. It is a TypeScript library and can be run within Node, such as when you run the sandbox. In the future it could be run inside wallet software or a browser.
+This page describes the Private Execution Environment (PXE), a client-side library for the execution of private operations. It is a TypeScript library that can be run within Node.js, inside wallet software or a browser.
 
-The PXE generates proofs of private function execution, and sends these proofs along with public function requests to the sequencer. Private inputs never leave the client-side PXE.
+The PXE is a client-side interface of the PXE Service. The PXE generates proofs of private function execution, and sends these proofs along with public function execution requests to the sequencer. Private inputs never leave the client-side PXE.
```

Comment:
```suggestion
The PXE generates proofs of private function execution, and sends these proofs along with public function execution requests to the sequencer. Private inputs never leave the client-side PXE.
```

Given there's no longer a server I'd just remove mentions of this, I don't think they add any value